src/runtime_src/core/tools/xbutil2/SubCmdExamine.cpp

```diff
@@ -172,6 +172,12 @@ SubCmdExamine::execute(const SubCmdOptions& _options) const
     throw xrt_core::error(std::errc::operation_canceled);
   }
 
+  // when json is specified, make sure an accompanying output file is also specified
+  if (!vm["format"].defaulted() && sOutput.empty()) {
```

Comment:
Should this code be as follows:
```suggestion
  if (!sFormat.empty() && sOutput.empty()) {
```